content/working_with/console/widgets/deploymentActionButtons.md

```diff
@@ -4,10 +4,9 @@ title: Deployment action buttons
 category: Widgets
 draft: false
 ---
-Buttons which allow running workflows of a specific deployment, updating it or deleting it. The deployment can be selected in one of the following ways:
+Buttons which allow running workflows of the deployment currently in the context, updating it or deleting it.
```

Comment:
I suspect you wanted to say:
```suggestion
Buttons which allow running workflows of the deployment currently set in the context, updating it or deleting it.
```
Am I right?

Reply:
My impression is that 'set' is not required there, but of course I have nothing against it - applying.